.gitlab-ci.yml

```diff
@@ -63,5 +63,6 @@ variables:
 
 # This is where jobs are included for each machine
 include:
-  - local: .gitlab/build_ruby.yml
+  # - local: .gitlab/build_ruby.yml
```

Comment:
Do we want to leave this around in case we want to move testing to Ruby @white238 ?

Reply:
Commented out is good for now since Ruby is currently down for the conversion.